Runtime/Tracking/Follow/Modifier/Property/Rotation/TransformPositionDifferenceRotation.cs

```diff
@@ -0,0 +1,98 @@
+﻿namespace Zinnia.Tracking.Follow.Modifier.Property.Rotation
+{
+    using UnityEngine;
+    using Malimbe.XmlDocumentationAttribute;
+    using Malimbe.PropertyValidationMethod;
+    using Malimbe.PropertySerializationAttribute;
+    using Zinnia.Extension;
+    using Zinnia.Data.Type;
+
+    /// <summary>
+    /// Updates the transform rotation of the target to match the difference in position of the source.
+    /// </summary>
+    public class TransformPositionDifferenceRotation : PropertyModifier
+    {
+        /// <summary>
+        /// The drag applied to the rotation.
```

Comment:
```suggestion
        /// The drag applied to the rotation to slow it down.
```
Just to prevent the user to lookup what "drag" is.